docs/internal_api.yml

```diff
@@ -99,6 +99,73 @@ paths:
           description: Internal server error.
           schema:
            $ref: "#/definitions/Error"
+  /deployments/{id}/deployments:
```

Comment:
did you mean `tenants/{id}....`?

Reply:
no, I didn't. 

The end point is: api/internal/v1/deployments/<id>/deployments